### ISSUE TITLE
newline missing in radclient stats

### ISSUE THREAD

**Issue by pauldekkers** [FreeRADIUS/freeradius-server#1928]
In `src/main/radclient.c`:

```
                printf("Packet summary:\n"
                       "\tAccepted      : %" PRIu64 "\n"
                       "\tRejected      : %" PRIu64 "\n"
                       "\tLost          : %" PRIu64 "\n"
                       "\tPassed filter : %" PRIu64 "\n"
                       "\tFailed filter : %" PRIu64,
```

I think Failed filter should have a newline too.

**Comment by pauldekkers** [comment]
Let's not be lazy ;-) I'll create a pull request for this!

**Comment by pauldekkers** [comment]
Fixed in 8962e549cc5d344263e5df3837121e69c64c6743 / #1929
